### PR TITLE
Use built-in virtual environment (not 3rd party)

### PR DESCRIPTION
**Pull request by treyhunner**
Python 3 comes with a `venv` package which allows you to create a virtual environment.

Asking users to use `virtualenv` presumes that they've installed the third-party `virtualenv` package. That package is necessary for using virtual environments in Python 2, but there isn't any benefit to using it (that I've ever seen at least) over the built-in Python 3 `venv` package.

I think it would probably be easiest for new users if we removed the requirement for `virtualenv` by using the built-in `venv`. This is how I usually teach Python 3 and this is what the Django Girls tutorial [recommends](https://tutorial.djangogirls.org/en/installation/#virtual-environment).

### CODE REVIEW

File: docs/internals/contributing.rst [hunk 1]
```diff
@@ -51,7 +51,7 @@ For Windows the use of cmd under Administrator permission is suggested instead o
 
 .. code-block:: batch
 
-    > virtualenv -p "C:\Python34\python.exe" env
+    > C:\Python34\python.exe -m venv env
```
Comment (goanpeca) [inline docs/internals/contributing.rst]: This assumes users will install on that location. This is not what windows10 offers as a default location.

Comment (treyhunner) [inline docs/internals/contributing.rst]: I agree that this is a problem. I was planning to make another pull request after this one to remove that full path and use the built-in `py -3` that comes when install Python on Windows.

Comment (treyhunner) [inline docs/internals/contributing.rst]: A full path is currently already assumed to be that location. So while still not fully accurate, I do think this is an improvement on what is currently in the tutorial.

**Comment by treyhunner** [comment]
I also changed the full path to `py -3` while I was at it. [That works on Windows](https://docs.python.org/3.4/using/windows.html?highlight=windows#from-the-command-line).

Re-review needed @freakboy3742.
